casper/src/main/scala/coop/rchain/casper/BlockCreator.scala

```diff
@@ -2,7 +2,7 @@ package coop.rchain.casper
 
 import cats.Monad
 import cats.effect.Sync
-import cats.syntax.all._
+import cats.implicits._
```

Comment:
Please use _syntax_  to import because _implicits_ includes much more things and type inference can have more ambiguities which requires more explicitly written types.